Gaurav.java

```diff
@@ -0,0 +1,30 @@
+import java.io.IOException;
+
+
+public class Gaurav {
+
+	 private static int count = 0;
+
+	
+	public static void main(String[] args) {
+		
+				System.out.println("This is Gaurav Garg testing using GIT");
+				
+				Process process =null;
+		
+	System.out.println("This is Gaurav Garg testing using GIT");		
+				try {
+					process.getOutputStream().close();
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') A "NullPointerException" could be thrown; "process" is nullable here. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://localhost:9000/coding_rules#rule_key=squid%3AS2259)